README.md

```diff
@@ -60,6 +60,7 @@ The current set of feature sources are the following:
 - Memory
 - Network
 - Storage
+- Selinux
```

Comment:
Please add some details related to this feature similar to https://github.com/kubernetes-incubator/node-feature-discovery#storage-features.